src/maps/hourly-parameter-fuel-flow.map.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@nestjs/common';
+import { BaseMap } from '@us-epa-camd/easey-common/maps';
+import { HrlyParamFuelFlow } from '../entities/hrly-param-fuel-flow.entity';
+import { HrlyParamFuelFlow as HrlyParamFuelFlowWorkspace } from '../entities/workspace/hrly-param-fuel-flow.entity';
+import { HourlyParamFuelFlowDTO } from '../dto/hourly-param-fuel-flow.dto';
+
+@Injectable()
+export class HourlyParameterFuelFlowMap extends BaseMap<
+  HrlyParamFuelFlow | HrlyParamFuelFlowWorkspace,
+  HourlyParamFuelFlowDTO
+> {
+  public async one(
+    entity: HrlyParamFuelFlow | HrlyParamFuelFlowWorkspace,
+  ): Promise<HourlyParamFuelFlowDTO> {
+    const monitoringSystemId = entity.monitorSystem?.monitoringSystemId ?? null;
+
+    return {
+      id: entity.id,
+      userId: entity.userId,
+      monitoringLocationId: entity.monitoringLocationId,
+      parameterCode: entity.parameterCode,
+      parameterValueForFuel: entity.parameterValueForFuel,
+      formulaIdentifier: entity.monitorFormula.formulaId,
```

Comment:
This value could potentially be null so need to add a check like for monitoringSystem